Dockerfile

```diff
@@ -49,6 +46,10 @@ RUN apt-get -qq update \
   && rm /tmp/sym-collector.zip \
   && chmod +x /usr/local/bin/SymbolCollector.Console
 
+RUN curl -fsSL https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.7.1-stable.tar.xz -o /tmp/flutter_linux_3.7.1-stable.tar.xz \
+  && tar xf /tmp/flutter_linux_3.7.1-stable.tar.xz \
+  && rm /tmp/flutter_linux_3.7.1-stable.tar.xz
```

Comment:
Using another layer due to OOM

Reply:
That should not be necessary or a solution, at least toy knowledge. Is there a source saying another layer would avoid an OOM?

Reply:
cloud build was OOMing while finalizing the layer

Reply:
Would be great if this is added as a comment in the file as it is definitely not obvious.